org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/java/org/eclipse/winery/bpmn2bpel/BPMN4Tosca2BpelTest.java

```diff
@@ -13,50 +13,178 @@
 	}
 
 	@Test
 	public void testTransformGateway()
-			throws ParseException, PlanWriterException, MalformedURLException, URISyntaxException {
+			throws ParseException,
+			PlanWriterException,
+			URISyntaxException, IOException {
 		URI srcUri = Paths.get(RESOURCES_DIR, "bpmn4tosca.exclusivegateway.json").toUri();
-		URI targetUri = Paths.get(RESOURCES_DIR, "managementplan.exclusivegateway.zip").toUri();
+		URI targetUri = Paths.get(RESOURCES_DIR, folder.newFile("managementplan.exclusivegateway.zip").getAbsolutePath()).toUri();
 		BPMN4Tosca2BpelTest.class.getResource(".");
 		Bpmn4Tosca2Bpel transformer = new Bpmn4Tosca2Bpel();
 		transformer.transform(srcUri, targetUri);
 	}
 
+
+	@Test(expected = ParseException.class)
+	public void testTransformExceptionMissingNode() throws ParseException,
+			PlanWriterException,
+			URISyntaxException, IOException {
+		URI srcUri = Paths.get(RESOURCES_DIR, "bpmn2tosca_exceptionNode.json").toUri();
+		URI targetUri = Paths.get(RESOURCES_DIR, folder.newFile("managementplan_exceptionNode.zip").getAbsolutePath()).toUri();
+		BPMN4Tosca2BpelTest.class.getResource(".");
+		Bpmn4Tosca2Bpel transformer = new Bpmn4Tosca2Bpel();
+		transformer.transform(srcUri, targetUri);
+		fail();
+	}
+
+	@Test(expected = ParseException.class)
+	public void testTransformExceptionMissingConnection() throws ParseException,
+			PlanWriterException,
+			URISyntaxException, IOException {
+		URI srcUri = Paths.get(RESOURCES_DIR, "bpmn2tosca_exceptionConnection.json").toUri();
+		URI targetUri = Paths.get(RESOURCES_DIR, folder.newFile("managementplan_exceptionConnection.zip").getAbsolutePath()).toUri();
+		BPMN4Tosca2BpelTest.class.getResource(".");
+		Bpmn4Tosca2Bpel transformer = new Bpmn4Tosca2Bpel();
+		transformer.transform(srcUri, targetUri);
+		fail();
+	}
+
+	@Test(expected = ParseException.class)
+	public void testTransformGatewayOneWayOut()
+			throws ParseException,
+			PlanWriterException,
+			URISyntaxException, IOException {
+		URI srcUri = Paths.get(RESOURCES_DIR, "bpmn4tosca.exclusivegatewayOneWayOut.json").toUri();
+		URI targetUri = Paths.get(RESOURCES_DIR, folder.newFile("managementplan.exclusivegateway.zip").getAbsolutePath()).toUri();
+		BPMN4Tosca2BpelTest.class.getResource(".");
+		Bpmn4Tosca2Bpel transformer = new Bpmn4Tosca2Bpel();
+		transformer.transform(srcUri, targetUri);
+		fail();
+
+	}
+
+	@Test(expected = ParseException.class)
+	public void testTransformExceptionTwoConnections() throws ParseException,
+			PlanWriterException,
+			URISyntaxException, IOException {
+		URI srcUri = Paths.get(RESOURCES_DIR, "bpmn2tosca_exceptionTwoConnections.json").toUri();
+		URI targetUri = Paths.get(RESOURCES_DIR, folder.newFile("managementplan_exceptionConnection.zip").getAbsolutePath()).toUri();
+		BPMN4Tosca2BpelTest.class.getResource(".");
+		Bpmn4Tosca2Bpel transformer = new Bpmn4Tosca2Bpel();
+		transformer.transform(srcUri, targetUri);
+		fail();
+	}
+
+	/**
+	 * @param archive1
```

Comment:
Please do not add empty parameters comments. Remove them. When pressing ctrl+alt+l in intellij, they will disappear.